Tutorials/pinpull.md

```diff
@@ -0,0 +1,49 @@
+# Pull- Pins
+
+Suppose a pin is configured as an input. If nothing is connected to the pin and the program tries to read the state of the pin, it would be in a 'floating' state i.e an unknown state. To prevent this, a pull-up or a pull-down state is defined. They are often used in the case of Buttons and Switches.
+
+Pins 2-7 on both the Ports are available for interrupts.
+
+Take a look at the following circuit. The code example given below turns on the Blue LED of the Tessel module when the pushbutton is pressed and turns off the Blue LED when the pushbutton is released.
+
+![Pin-Pull using Push buttons](http://i.imgur.com/OYJZ8Dp.jpg)
+
+```js
+var tessel= require('tessel'); // Import Tessel
+
+var pin = tessel.port.A.pin[2]; // Select pin 2 on port A
+
+var pullType = "pullup"; // Set the mode of `pin.pull to pullup`
```

Comment:
"pullup" is the default arg value as defined in the t2-firmware code here : https://github.com/tessel/t2-firmware/blob/master/node/tessel-export.js#L737-L758